cypress/integration/spec.js

```diff
@@ -0,0 +1,34 @@
+/// <reference types="Cypress" />
+
+import { SearchPage } from "../page-objects/search-page"
+
+// @ts-check
+const searchPage = new SearchPage()
+describe('Article search', () =>{
+  beforeEach(() =>{
+    searchPage.navigateToSearchPage()
+    searchPage.search("catt")
+  })
+
+  it('search should show results', () => {
+    cy.get('.result').first()
+      .should('have.text', "Cattaraugus County, New YorkCounty in New York")
+  })
+
+  it('results with image should show image', () => {
+    cy.get('.result').first()
+      .children().should('have.length', 2)
```

Comment:
The exact number of children is something that can change for unrelated reasons. For instance: it is common to change the HTML structure to accomplish different alignment requirements. 

A more stable way to test could be to check the presence of an <img> tag inside the search result row.